.github/workflows/cxx11-ubuntu.yaml

```
@@ -16,8 +16,12 @@ jobs:
     - name: configure
       run: |
         mkdir _build && cd _build
-        cmake ../ -DENABLE_STDCXX_SYNC=ON -DENABLE_UNITTESTS=ON -DENABLE_BONDING=ON -DENABLE_TESTING=ON -DENABLE_EXAMPLES=ON
+        cmake ../ -DENABLE_STDCXX_SYNC=ON -DENABLE_ENCRYPTION=ON -DENABLE_UNITTESTS=ON -DENABLE_BONDING=ON -DENABLE_TESTING=ON -DENABLE_EXAMPLES=ON -DENABLE_CODE_COVERAGE=ON
     - name: build
       run: cd _build && cmake --build ./
     - name: test
-      run: cd _build && ctest --extra-verbose
+      run: |
+        cd _build && ctest --extra-verbose
+        cd ../
+        source ./scripts/collect-gcov.sh
+        bash <(curl -s https://codecov.io/bash)
```

I wonder if it would work this way.
```suggestion
      run: |
        cd _build && ctest --extra-verbose
    - name: codecov
      run: |
        source ./scripts/collect-gcov.sh
        bash <(curl -s https://codecov.io/bash)
```

---

This seems to be working just fine on my fork